pages/form/WorkOnSiteDays.jsx

```diff
@@ -40,18 +40,28 @@ export default function WorkOnSiteDays() {
   }
 
   // we  pass this function as props to our child component to update Form data and logs
-  const saveDataAndShowLog = () => {
+  const saveDataAndShowLog = (msg="Next button clicked") => {
     saveAnswers();
-    sendLogs(logMessage("Next button clicked"));
+    sendLogs(logMessage(msg));
+  }
+
+  const backButtonOnClickHandler = (msg="Back button clicked") => {
+    saveDataAndShowLog(msg)
+    history.back();
   }
 
   return (
     <Layout isText={true} Progress={Q3Progress}>
       <Box pos="absolute" top={["2", "5"]} left={["2", "10"]}>
-        <BackButton
-          href="/"
-          onClick={saveDataAndShowLog}
-        />
+        <Button
+          onClick={() => backButtonOnClickHandler()}
+          variant="ghost"
+        >
+          <BackArrow />
+          <Text ms={2} as="u" color="var(--chakra-colors-blue-600)">
+            {"Back"}
+          </Text>
+        </Button>
```

Comment:
replace `<BackButton>` component with regular Chakra `<Button>` component:
- we no longer have to wrap `<Button>` component inside a `<Link>` component